target/linux/lantiq/patches-4.19/0310-NAND-add-easybox904-bbt-byDTS.patch

```diff
@@ -0,0 +1,174 @@
+From d96bdf6c85f2d02cb9dcb6c71d1b510fd9e09199 Mon Sep 17 00:00:00 2001
+From: Jonas Albrecht <plonkbong100@protonmail.com>
+Date: Sun, 19 Jan 2020 17:31:43 +0000
+Subject: [PATCH] mtd: nand_bbt: BBT impl. for Samsung K9F4G08U0x
+X-Patchwork-Bot: notify
+
```

Comment:
How to test that i really need this? Like dump the flash and look for those blocks and whether they got used where they should not.

And is that signature something the controller puts on those blocks or is it pure software?

If it is pure software me might get away with sort of reformatting the whole flash and go back to the driver default. Which might require a custom bootloader after all, or touching all partitions. Touching all partitions, or all that owrt will ever write i guess.